src/solver/simulation/include/antares/solver/simulation/solver.hxx

```diff
@@ -523,19 +540,19 @@ uint ISimulation<ImplementationType>::buildSetsOfParallelYears(
         // created
         bool refreshing = false;
         refreshing = pData.haveToRefreshTSLoad && (y % pData.refreshIntervalLoad == 0);
-        refreshing
-          = refreshing || (pData.haveToRefreshTSSolar && (y % pData.refreshIntervalSolar == 0));
-        refreshing
-          = refreshing || (pData.haveToRefreshTSWind && (y % pData.refreshIntervalWind == 0));
-        refreshing
-          = refreshing || (pData.haveToRefreshTSHydro && (y % pData.refreshIntervalHydro == 0));
+        refreshing = refreshing
```

Comment:
operator || at end of line rather than at the beginning ? (so that each choice starts at the same column)

Reply:
Why not, again there is only one parameter for all operators (including assignment), so it will change quite a lot of code.